r2/r2/templates/preffeeds.html

```diff
@@ -127,5 +127,15 @@
     </td>
   </tr>
   %endif
+  <tr>
+    <th>your friends</th>
+    <td class="prefright">
+      <%self:feedbuttons path="/friends"></%self:feedbuttons>
```

Comment:
Shouldn't these links be [`/r/friends`](https://reddit.com/r/friends) and [`/r/friends/comments`](https://reddit.com/r/friends/comments)?  [`/friends`](https://reddit.com/friends) and [`/friends/comments`](https://reddit.com/friends/comments) don't seem to be valid pages.